build/cqs.tsv

```diff
@@ -27,23 +26,22 @@ com.googlecode.efficient-java-matrix-library:core	0.26	compile
 com.google.code.gson:gson	2.8.1	compile
 com.googlecode.json-simple:json-simple	1.1.1	compile
 com.googlecode.juniversalchardet:juniversalchardet	1.0.3	compile
+com.google.errorprone:error_prone_annotations	2.3.3	compile
+com.google.guava:guava	11.0.2	compile
```

Comment:
*blinks*  How did Guava 11 make it back in?!?!?!

Reply:
that's the version used by hadoop 2.8...

Reply:
~https://dev.eclipse.org/ipzilla/show_bug.cgi?id=8508~